### PR TITLE
chore(docs): adding info box for compatibility info (#927)

### PR DESCRIPTION
**Pull request by amyshwang**
Adding info to two blog posts to reflect versions used in the tutorial, as newer versions no longer work with QuestDB.

### CODE REVIEW

File: blog/2021-11-01-plotly-finnhub-realtime-dashboard.mdx [hunk 1]
```diff
@@ -30,6 +30,14 @@ This post comes from Gábor Boros, who has written an excellent tutorial that
 shows how to build dashboards using Plotly and QuestDB that track and chart
 stock prices in real-time. Thanks for the submission, Gábor!
 
+
+:::note
+
+This tutorial is based on SAQLAlchemy version 1.4.21. 
+Later versions of SQLAlchemy may not work for this tutorial.
```
Comment (pswu11) [inline blog/2021-11-01-plotly-finnhub-realtime-dashboard.mdx:37]: I'm pretty sure I'm nit-picking. 

This tutorial ...
... for this tutorial 
seems repetitive. 

Perhaps this is enough. 
Later versions of SQLAlchemy may not work.

Comment (amyshwang) [reply blog/2021-11-01-plotly-finnhub-realtime-dashboard.mdx:37]:
```suggestion
Later versions of SQLAlchemy may not work.
```

Comment (amyshwang) [reply blog/2021-11-01-plotly-finnhub-realtime-dashboard.mdx:37]: Thanks - I think you are very right!

**Review by pswu11 — APPROVED** [review]
LGTM

**Pull request merged**